.yarn/versions/8be28082.yml

```diff
@@ -0,0 +1,7 @@
+releases:
```

Comment:
This isn't a big deal that needs resolving, but I would try to be a little more granular with the versioning to better reflect actual changes that happened in each package. The aim is to align with semantic versioning:
- major = breaking change
- minor = new API addition, still backwards compatible with major version
- patch = fixes that don't affect the API

In this case, I would suggest that arquero-react is a minor, and everything else would be "decline" because the tests are only for tests and docs that don't need released.